Dockerfile

```diff
@@ -0,0 +1,45 @@
+FROM alpine:3.9
+
+LABEL maintainer="mritd <mritd1234@gmail.com>"
+
+ARG TZ='Asia/Shanghai'
+
+ENV TZ ${TZ}
+ENV LINUX_HEADERS_DOWNLOAD_URL=http://dl-cdn.alpinelinux.org/alpine/v3.7/main/x86_64/linux-headers-4.4.6-r2.apk
+
+COPY . /trojan
+
+WORKDIR /trojan
+
+RUN set -ex \
+    && apk upgrade \
+    && apk add bash tzdata\
+    && apk add --virtual .build-deps \
+        build-base \
+        boost-dev \
+        curl \
+        cmake \
+        mariadb-dev \
+        tar \
+        git \
+    && curl -sSL ${LINUX_HEADERS_DOWNLOAD_URL} > /linux-headers-4.4.6-r2.apk \
+    && apk add --virtual .build-deps-kernel /linux-headers-4.4.6-r2.apk \
```

Comment:
Use a lower version of the kernel header for TFO to be compatible with most docker host systems.

refs: mritd/dockerfile#73